evap/staff/templates/staff_semester_view_evaluation.html

```diff
@@ -187,6 +187,25 @@
             <a href="{% url 'staff:evaluation_copy' evaluation.id %}" class="btn btn-sm btn-light" data-bs-toggle="tooltip" title="{% translate 'Copy' %}">
                 <span class="fas fa-copy"></span>
             </a>
+            {% if not evaluation.has_exam_evaluation %}
+                <confirmation-modal type="submit" name="evaluation_id" value="{{ evaluation.id }}" confirm-button-class="btn-primary" form="exam_creation_form_{{ evaluation.id }}">
+                    <span slot="title">{% translate 'Create exam evaluation' %}</span>
+                    <span slot="action-text">{% translate 'Create exam evaluation' %}</span>
+                    <span slot="question">
+                        {% blocktranslate trimmed %}
+                            Create an exam evaluation based on this evaluation. This will copy all the participants and contributors from the original evaluation. It will set the weight of the original evaluation to 9 and its end date will be set to the day before the exam.
+                        {% endblocktranslate %}
+                    </span>
+                    <div slot="extra-inputs">
+                        <label>
+                            {% translate 'Exam Date:' %}
+                            <input type="date" name="exam_date" data-earliest-possible-date="{{ evaluation.earliest_possible_exam_date }}" data-earliest-possible-date-reason="{% translate 'The end date of the main evaluation would be before its start date.' %}" class="form-control" required form="exam_creation_form_{{ evaluation.id }}"/>
```

Comment:
I just found out: We can directly add a `min` field here, so that the user is not even tempted to select invalid values, as described in https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date. `data-earliest-possible-date-reason` would then become maybe obsolete